WordPress-Theme/ruleset.xml

```diff
@@ -89,6 +89,12 @@
 	<!-- Prohibit the use of the backtick operator. -->
 	<rule ref="Generic.PHP.BacktickOperator"/>
 
+	<!-- Discourage a number of functions for usage in a theme. -->
+	<rule ref="WordPress.PHP.DiscouragedFunctions"/>
```

Comment:
This sniff is now deprecated.

The `WordPress.PHP.PHPDiscouragedFunctions` sniff covers at least part of it.